e2e/rbd.go

```diff
@@ -1691,111 +1691,144 @@ var _ = Describe("RBD", func() {
 				}
 			})
 
-			By("validate PVC mounting if snapshot and parent PVC are deleted", func() {
-				// snapshot beta is only supported from v1.17+
-				if !k8sVersionGreaterEquals(f.ClientSet, 1, 17) {
-					Skip("pvc restore is only supported from v1.17+")
-				}
-				err := createRBDSnapshotClass(f)
-				if err != nil {
-					e2elog.Failf("failed to create storageclass with error %v", err)
-				}
-				defer func() {
-					err = deleteRBDSnapshotClass()
+			By(
+				"validate PVC mounting if snapshot and parent PVC are deleted chained with depth 2",
+				func() {
+					// snapshot beta is only supported from v1.17+
+					if !k8sVersionGreaterEquals(f.ClientSet, 1, 17) {
+						Skip("pvc restore is only supported from v1.17+")
+					}
+					snapChainDepth := 2
+
+					err := deleteResource(rbdExamplePath + "storageclass.yaml")
 					if err != nil {
-						e2elog.Failf("failed to delete VolumeSnapshotClass: %v", err)
+						e2elog.Failf("failed to delete storageclass with error %v", err)
 					}
-				}()
 
-				// create PVC and bind it to an app
-				pvc, err := loadPVC(pvcPath)
-				if err != nil {
-					e2elog.Failf("failed to load PVC with error %v", err)
-				}
+					err = createRBDStorageClass(
+						f.ClientSet,
+						f,
+						defaultSCName,
+						nil,
+						map[string]string{
```

Comment:
There is no test without encryption anymore? If encryption is needed to be tested with this combination, a new test should be added.